netjsonconfig/schema.py

```diff
@@ -443,11 +451,83 @@
                 }
             }
         },
+        "encryption_mfp_property": {
+            "properties": {
+                "ieee80211w": {
+                    "type": "string",
+                    "title": "management frame protection",
+                    "enum": ["0", "1", "2"],
+                    "options": {"enum_titles": ["disabled", "optional", "required"]},
+                    "propertyOrder": 4,
+                }
+            }
+        },
+        "encryption_mfp_property_required": {
+            "required": ["ieee80211w"],
+            "properties": {
+                "ieee80211w": {
+                    "type": "string",
+                    "title": "management frame protection",
+                    "enum": ["2"],
+                    "options": {"enum_titles": ["required"]},
+                    "propertyOrder": 4,
+                }
+            },
+        },
+        "encryption_mfp_property_optional": {
+            "required": ["ieee80211w"],
+            "properties": {
+                "ieee80211w": {
+                    "type": "string",
+                    "title": "management frame protection",
+                    "enum": ["1", "2"],
+                    "options": {"enum_titles": ["optional", "required"]},
+                    "propertyOrder": 4,
+                }
+            },
+        },
+        "encryption_wpa3_personal": {
+            "title": "WPA3 only Personal",
+            "allOf": [
+                {"$ref": "#/definitions/encryption_base_settings"},
+                {"$ref": "#/definitions/encryption_cipher_property"},
+                {"$ref": "#/definitions/encryption_mfp_property_required"},
+                {
+                    "properties": {
+                        "protocol": {
+                            "enum": ["wpa3_personal"],
+                            "options": {"enum_titles": ["WPA3 Personal"]},
+                        },
+                        "key": {"minLength": 8},
+                    }
+                },
+            ],
+        },
+        "encryption_wpa3_2_personal": {
+            "title": "WPA3/WPA2 Personal",
```

Comment:
I think we can make it explicit here that it is mixed mode. 
```suggestion
            "title": "WPA3/WPA2 Personal Mixed Mode",
```